src/backend/src/lib.rs

```diff
@@ -265,6 +266,12 @@ async fn caller_btc_address(network: BitcoinNetwork) -> String {
     public_key_to_p2pkh_address(network, &ecdsa_pubkey_of(&ic_cdk::caller()).await)
 }
 
+/// Returns the balance of the given Bitcoin address.
+#[query]
```

Comment:
Maybe a stupid question: Query is correct? `bitcoin_get_balance` isn't an inter-canister update call?